R/TealAppDriver.R

```diff
@@ -645,6 +647,28 @@ TealAppDriver <- R6::R6Class( # nolint: object_name.
       }
 
       NULL # If there are not any supported filters
+    },
+    #' @description
+    #' Check if the page is stable without any `DOM` updates in the body of the app.
+    #' This is achieved by blocing the R process by sleeping until the page is unchanged till the `stability_period`.
+    #' @param stability_period (`numeric(1)`) The time in milliseconds to wait till the page to be stable.
+    #' @param check_interval (`numeric(1)`) The time in milliseconds to check for changes in the page.
+    #' The stability check is reset when a change is detected in the page after sleeping for check_interval.
```

Comment:
```suggestion
    # @description
    # Check if the page is stable without any `DOM` updates in the body of the app.
    # This is achieved by blocing the R process by sleeping until the page is unchanged till the `stability_period`.
    # @param stability_period (`numeric(1)`) The time in milliseconds to wait till the page to be stable.
    # @param check_interval (`numeric(1)`) The time in milliseconds to check for changes in the page.
    # The stability check is reset when a change is detected in the page after sleeping for check_interval.
```

Reply:
Having roxygen2 comments here is not going to help with documentation as R6 class methods are not documented by roxygen2. But, it is better for readability in the code and all other methods use this. I see that we've missed it in a private method `get_active_filter_selection` so I'll add them there as well. Good catch.

Reply:
Good catch, I'll remove them

Reply:
I just learned that for private methods roxygen comments do not work.

Reply:
Interesting that our roxygen doc CI is unable to catch this